web/source/build.sh

```diff
@@ -457,7 +457,17 @@ fi
 
 ### -embed section complete.
 
+### TEMP - to link @keymanapp/gesture-recognizer in for CI while detached
 if [ $BUILD_COREWEB = true ]; then
+    echo "Compiling the detached @keymanapp/gesture-recognizer module"
+    pushd "$KEYMAN_ROOT/common/web/gesture-recognizer/src"
+    ./build.sh build || fail "@keymanapp/gesture-recognizer build was unsuccessful."
+    popd
```

Comment:
The child script already sets its folder, so let's simplify (this change also assumes build.sh moved to component root):

```suggestion
    "$KEYMAN_ROOT/common/web/gesture-recognizer/build.sh" build
```

We shouldn't need `|| fail` either because `set -e`.